Casks/a/arq.rb

```diff
@@ -15,6 +15,7 @@
   auto_updates true
 
   pkg "Arq#{version}.pkg"
+  binary "#{appdir}/Arq.app/Contents/Resources/arqc
```

Comment:
Missing a closing quote

```suggestion
  binary "#{appdir}/Arq.app/Contents/Resources/arqc"
```